scripts/generate_audible_export.py

```
@@ -399,7 +399,7 @@ def generate_upload_instructions(
         f"{album}",
         f"",
         f"### Subtitle",
-        f"The Complete Idiot's Guide to Legally Bribing Your Way to Utopia",
+        f"A Step-by-Step Guide to Optimizing Your Terrible Civilization",
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)